db/migrate/20171124131432_create_registries.rb

```diff
@@ -1,6 +1,6 @@
-class CreateDockerRegistries < ActiveRecord::Migration
+class CreateRegistries < ActiveRecord::Migration
   def change
-    create_table :docker_registries do |t|
+    create_table :registries do |t|
```

Comment:
Shouldn't this be done in a new migration? Isn't it going to break customers upgrading from 2.0?

Reply:
Unless we didn't provide a maintenance to 2.0 yet.

Reply:
we dont have a registry model in 2.0, so i assumed i can still just update the migration

Reply:
But we were going to provide a maintenance update because this suse registry feature was needed by the CAP team, no? I'm just not sure if we still have time to make the change like this or not. If an update was already released, then we can't just change the name here.

Reply:
afaics it shouldnt affect the 2.0 code as we're writing directly to pillar, w/o any registry model